pyomo/contrib/pynumero/interfaces/pyomo_grey_box_nlp.py

```diff
@@ -426,7 +426,11 @@ def load_state_into_pyomo(self, bound_multipliers=None):
         # not we are minimizing or maximizing - this is done in the ASL interface
         # for ipopt, but does not appear to be done in cyipopt.
         obj_sign = 1.0
-        objs = list(m.component_data_objects(ctype=pyo.Objective, descend_into=True))
+        objs = list(
+            m.component_data_objects(
+                ctype=pyo.Objective, active=True, descend_into=True
```

Comment:
Leave a comment on why are we adding `active=True` here and at least let's run this through @michaelbynum to see if it is a good change